src/Tribe/Main.php

```diff
@@ -26,17 +34,10 @@ class Tribe__Events__Main {
 
 		const VERSION           = '3.12.3';
 		const MIN_ADDON_VERSION = '3.12';
-		const FEED_URL          = 'https://theeventscalendar.com/feed/';
```

Comment:
Sure we don't need this? We still refer to it on [line 3901](https://github.com/moderntribe/the-events-calendar/blob/feature/standalone-tickets-framework/src/Tribe/Main.php#L3901) (the `outputDashboardWidget()` method) - that will cause a fatal so we should change that method or else add it back as appropriate.